make/scripts/compare.sh

```diff
@@ -76,26 +76,13 @@ fi
 # disassembly, such as hard-coded addresses, to be able to catch "actual" differences.
 
 if [ "$OPENJDK_TARGET_OS" = "windows" ]; then
-  if [ "$OPENJDK_TARGET_CPU" = "x86" ]; then
-    DIS_DIFF_FILTER="$SED -r \
-        -e 's/^  [0-9A-F]{16}: //' \
-        -e 's/^  [0-9A-F]{8}: /  <ADDR>: /' \
-        -e 's/(offset \?\?)_C@_.*/\1<SYM>/' \
-        -e 's/[@?][A-Za-z0-9_]{1,25}/<SYM>/' \
-        -e 's/([-,+])[0-9A-F]{2,16}/\1<HEXSTR>/g' \
-        -e 's/\[[0-9A-F]{4,16}h\]/[<HEXSTR>]/' \
-        -e 's/: ([a-z]{2}[a-z ]{2})        [0-9A-F]{2,16}h?$/: \1        <HEXSTR>/' \
-        -e 's/_20[0-9]{2}_[0-1][0-9]_[0-9]{2}/_<DATE>/' \
-        "
-  elif [ "$OPENJDK_TARGET_CPU" = "x86_64" ]; then
-    DIS_DIFF_FILTER="$SED -r \
-        -e 's/^  [0-9A-F]{16}: //' \
-        -e 's/\[[0-9A-F]{4,16}h\]/[<HEXSTR>]/' \
-        -e 's/([,+])[0-9A-F]{2,16}h/\1<HEXSTR>/' \
-        -e 's/([a-z]{2}[a-z ]{2})        [0-9A-F]{4,16}$/\1        <HEXSTR>/' \
-        -e 's/\[\?\?_C@_.*/[<SYM>]/' \
-        "
-  fi
+  DIS_DIFF_FILTER="$SED -r \
```

Comment:
No, it was not intentional, as in I forgot about the aarch64 version of Windows.

With that said, I think it still might make sense to keep it this way. I don't think anyone has ever tried running the compare script on windows-aarch64; if they had, the lack of a filter at all would have made it basically unusable. This pattern is trying to hide 64-bit hex strings, and it is reasonable to assume it will work for aarch64 as well. If it doesn't, then its better to use this as a starting point for tweaking. Good catch, though!